cmd/end2endtest/helpers.go

```diff
@@ -393,6 +441,76 @@ func (t *e2eElection) sendVote(v voteInfo, apiClientMtx *sync.Mutex) (int, error
 	return contextDeadline, nil
 }
 
+// ballotVotes allow to create random vote values that possibly exceed the max value, max total cost and is not unique
+func (t *e2eElection) ballotVotes() ([][]int, [][]*types.BigInt) {
```

Comment:
i have the feeling this is not generic enough to belong to e2eElection, isn't it something specific of E2EBallotElection?

Reply:
Yeah, agree. It could be just a function

Reply:
i'd rather meant to move this out of helpers.go , into ballot.go